app/code/Codifi/Training/Logger/Handler.php

```diff
@@ -22,12 +24,12 @@ class Handler extends Base
      *
      * @var int
      */
-    protected $loggerType = Logger::ERROR;
+    public $loggerType = Logger::ERROR;
```

Comment:
We can't change visibility in child class from protected to public (possibly to private or leave as is in case of inheritance).